convert2rhel/unit_tests/pkghandler_test.py

```diff
@@ -155,6 +155,10 @@ def test_call_yum_cmd(self):
         self.assertEqual(utils.run_subprocess.cmd, "yum install -y")
 
     @unit_tests.mock(pkghandler, "call_yum_cmd", CallYumCmdMocked())
+    @unittest.skipIf(
+        not get_os_release()["NAME"].startswith("CentOS"),
+        reason="This test assumed to be run on CentOS system only."
```

Comment:
Actually, the test would run fine on CentOS, Oracle Linux and RHEL. I was brave enough to try now and they run on Fedora as well, all pass.
So I'd create another function like `is_rpm_based_os()` where you'd be comparing the what we call [system id](https://github.com/oamg/convert2rhel/blob/main/convert2rhel/systeminfo.py#L74) in convert2rhel (just a lower case first word of `/etc/system-release`) to oracle/centos/red/fedora, and use this function to skip the test if not on rpm bases system.